src/detectChildProcessRule.ts

```diff
@@ -0,0 +1,128 @@
+import * as Lint from 'tslint';
+import * as tsutils from 'tsutils';
+import * as ts from 'typescript';
+import { AstUtils } from './utils/AstUtils';
+
+import { ExtendedMetadata } from './utils/ExtendedMetadata';
+
+const FORBIDDEN_IMPORT_FAILURE_STRING: string = 'Found child_process import';
+const FOUND_EXEC_FAILURE_STRING: string = 'Found child_process.exec() with non-literal first argument';
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static metadata: ExtendedMetadata = {
```

Comment:
This should also have a `rationale`, which will be particularly helpful once rule documentation is auto-generated. You can generally copy & paste the rough description from the README.md table.

Reply:
Fixed